docs/DeveloperGuide.md

```diff
@@ -133,85 +134,50 @@ Classes used by multiple components are in the `seedu.patientbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### \[Proposed\] Undo/redo feature
+### 1. Done feature
+The mark done feature allows users to mark a specific appointment in the address book as done using `d/` tag and `t/` tag to specify the appointment at a certain time slot.
+#### 1.1 Implementation
+Command: `done d/DATE t/TIME`
 
-#### Proposed Implementation
+Example Usage: 
+* `done d/Monday t/9am`
+* `done d/12-12-2020 t/12pm`
 
-The proposed undo/redo mechanism is facilitated by `VersionedPatientBook`. It extends `PatientBook` with an undo/redo history, stored internally as an `patientBookStateList` and `currentStatePointer`. Additionally, it implements the following operations:
+The Mark Appointments as Done Feature is facilitated by the `DoneCommand`, which extends the abstract class `Command`, and
+the `DoneCommandParser`, which implements the `Parser` interface. Both of these classes are part of the `Logic` component.
+Additionally, the command takes in a `DateTimeLoader`, which contains the date and time information of the appointment to mark.
 
-* `VersionedPatientBook#commit()` — Saves the current patient book state in its history.
-* `VersionedPatientBook#undo()` — Restores the previous patient book state from its history.
-* `VersionedPatientBook#redo()` — Restores a previously undone patient book state from its history.
+The following is an example usage scenario and how the mark as done mechanism behaves at each step:
```

Comment:
Perhaps you may consider the following phrase instead "Please refer to the following for an example usage scenario on how the mark as done mechanism works in each step:". 

Let me know what you think! 😄